samples/nrf_desktop/src/modules/usb_state.c

```diff
@@ -239,15 +239,15 @@ static void device_status(enum usb_dc_status_code cb_status, const u8_t *param)
 		break;
 
 	case USB_DC_CONFIGURED:
-		__ASSERT_NO_MSG(state == USB_STATE_POWERED);
+		__ASSERT_NO_MSG(state != USB_STATE_DISCONNECTED);
 		new_state = USB_STATE_ACTIVE;
 		break;
 
 	case USB_DC_RESET:
 		if (state == USB_STATE_DISCONNECTED) {
```

Comment:
I think this can be asserted now (could you try?)

Reply:
Assertion is now fine, i.e. it is not hit in normal conditions.